pkg/reconciler/contour/lister.go

```diff
@@ -43,7 +43,15 @@ var _ status.ProbeTargetLister = (*lister)(nil)
 func (l *lister) ListProbeTargets(ctx context.Context, ing *v1alpha1.Ingress) ([]status.ProbeTarget, error) {
 	var results []status.ProbeTarget
 
-	visibilityKeys := config.FromContext(ctx).Contour.VisibilityKeys
+	cfg := config.FromContext(ctx)
+
+	port, scheme := int32(80), "http"
+	switch cfg.Network.HTTPProtocol {
+	case network.HTTPDisabled, network.HTTPRedirected:
```

Comment:
Can you add test case for this configuration?

Reply:
Done.